packages/editor/src/lib/editor/Editor.ts

```diff
@@ -8751,6 +8771,258 @@ export class Editor extends EventEmitter<TLEventMap> {
 		return this
 	}
 
+	private _zoomToFitPageContentAt100Percent() {
+		const bounds = this.getCurrentPageBounds()
+		const targetZoom = this.getBaseZoom()
+		if (!bounds) {
+			this.setCamera({ x: 0, y: 0, z: targetZoom }, { immediate: true })
```

Comment:
i should remove this, it's only valid on the initial navigation action, and I think this is the initial camera anyway, more or less.